model/websocket_client.go

```diff
@@ -20,15 +20,15 @@ const (
 // communicate with a websocket endpoint.
 type WebSocketClient struct {
 	Url                string                  // The location of the server like "ws://localhost:8065"
-	ApiUrl             string                  // The api location of the server like "ws://localhost:8065/api/v3"
+	ApiUrl             string                  // The Api location of the server like "ws://localhost:8065/api/v3"
```

Comment:
I was suggesting to use the all uppercase form `API`, same way as we do with `URL` below.
I know there's some inconsistency with the variable naming but we can try to make it right in the docs.